pipeline/report.py

```diff
@@ -388,6 +388,20 @@ def make(self, key):
         plt.close('all')
         self.insert1({**key, **fig_dict})
 
+
+@schema
+class ProbeLevelDriftMap(dj.Computed):
+    definition = """
+    -> ephys.ProbeInsertion
+    -> ephys.ClusteringMethod
+    ---
+    driftmap: filepath@report_store
+    """
+
+    def make(self, key):
+        pass
```

Comment:
guessing this + shell/worker hooks are planned for future PR?

Reply:
yes, place holder for now, will implement next